src/ontology/edits/oeo-shared.omn

```diff
@@ -131,18 +131,87 @@ ObjectProperty: <http://purl.obolibrary.org/obo/IAO_0000136>
     
 ObjectProperty: <http://purl.obolibrary.org/obo/RO_0000053>
 
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000118> "has characteristic"
+    
+    Range: 
+        <http://purl.obolibrary.org/obo/BFO_0000020>
+    
     InverseOf: 
         OEO_00010121
     
     
+ObjectProperty: <http://purl.obolibrary.org/obo/RO_0000056>
+
+    Domain: 
+        <http://purl.obolibrary.org/obo/BFO_0000002>
+    
+    Range: 
+        <http://purl.obolibrary.org/obo/BFO_0000003>
```

Comment:
The definition is: _a relation between a continuant and a **process**, in which the continuant is somehow involved in the process_. Shouldn't the range be then `process` instead of `occurant`?